doc/models.md

```diff
@@ -14,7 +14,7 @@
 - Thumbnail image available
 - Layer-related data is named and structured 
   - either by [workspace](#workspace) name and layername
-      - [REST API](rest.md): `/rest/workspaces/<workspace_name>/layers/<layername>` 
+      - [REST API](rest.md): `/rest/layers/<uuid>/thumbnail` 
```

Comment:
```suggestion
      - [REST API](rest.md): `/rest/workspaces/<workspace_name>/layers/<layername>`
```

Keep the original line unchanged (because some workspace-layer endpoints already exist)